integration_tests/broker_test.go

```diff
@@ -87,13 +158,16 @@ var _ = Describe("Broker", func() {
 			uriParam{key: "plan_id", value: planID},
 			uriParam{key: "accepts_incomplete", value: strconv.FormatBool(acceptsIncomplete)},
 		)
-		resp := serveRequest(responseRecorder, req)
-		Expect(responseRecorder.Code).To(Equal(http.StatusInternalServerError))
-		body := readResponseBody(resp)
-		Expect(string(body)).To(ContainSubstring("Can't deprovision an instance"))
+		brokerAPI.ServeHTTP(responseRecorder, req)
+
+		foundDeployments := *fakeComposeClient.FoundDeployments
+		Expect(fakeComposeClient.DeprovisionDeploymentID).To(Equal(foundDeployments[1].ID))
```

Comment:
I'm not very clear how this relates to the `instanceID` that's passed in with the request.